CONTRIBUTING.rst

```diff
@@ -11,7 +11,11 @@ before contributing.
 Meetings
 ========
 
-* `Community Meetings <https://hackmd.io/sSB1pwpDR5Seag0YB-vYMA>`_
+* `Public Calendar RSTUF Meetings <https://calendar.google.com/calendar/u/0?cid=Y19hYWFjYjc2M2NkNTliNWJhOWUyYmY4N2U1MTJhM2Q4ZjEyYjkxNmFmYzdhOWM4YjQxMmZmNjcwZWYzNmFiOTdlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20>`_
+
+  - `Community Meetings <https://docs.google.com/document/d/13a_AtFpPK9WO4PlAN6ciD-G1jiBU3gEDtRD1OUinUFY>`_
```

Comment:
Is it better if we use a platform not connected to a Google account?
Where is this google doc hosted?
Will we have access if something happens with that Google account? 
Is `HackMD` better for that purpose?

Reply:
For community meetings, we can expect non-technical people to edit this file.
About Google Docs, I don't see a problem. Anyone can edit this file (we made it public), and a new copy can be made.
This is used by OpenSSF as well.